source/styles.css

```diff
@@ -1,16 +1,17 @@
 .VirtualSelectGrid {
   z-index: 1;
 }
-
 .VirtualizedSelectOption {
   display: -webkit-box;
   display: -ms-flexbox;
   display: flex;
   -webkit-box-align: center;
-      -ms-flex-align: center;
-          align-items: center;
+  -ms-flex-align: center;
+  align-items: center;
   padding: 0 .5rem;
   cursor: pointer;
+  height: auto;
```

Comment:
This is the default value. Why specify it explicitly?

Reply:
This should also be overridden by react-virtualized's inline `style` (which has a fixed height)